tests/db-tests.test.ts

```diff
@@ -0,0 +1,110 @@
+import { BenchmarkData, Criterion, DataPoint } from '../src/api.js';
```

Comment:
I was think of something much simpler, and more targeted.

Could we perhaps add tests that really just test the `db.recordMeasurment*` methods?

This could go into the `db-setup.test.ts` file, possible as a separate `describe` with 3 or so `it` bits. Maybe right after the `describe('Setup of PostgreSQL DB'` and before `Recording a ReBench execution data fragment`.

Your approach duplicates quite a bit of code, which isn't ideal.
This includes the duplication of the `INSERT` statement, instead of testing the one from `db.ts`.